pkg/lvgl/Makefile

```diff
@@ -1,18 +1,18 @@
 PKG_NAME=lvgl
 PKG_URL=https://github.com/littlevgl/lvgl
-PKG_VERSION=2ead4959038fcfd01eaee0f124b98a8ae3efec21 # v6.1.2
+PKG_VERSION=57079452b0da9b9072fa88da8354e020cc6b6e6b # v7.2.0
```

Comment:
```suggestion
PKG_VERSION=eb76519d254ef6a55077e973c42ada2cd3e6666a # v7.3.0
```

Reply:
There's indeed a tag but the latest release is still 7.2 (https://github.com/lvgl/lvgl/releases). I can wait a couple of days and see if they release 7.3.

Reply:
7.3 has been released yesterday :)

Reply:
The version is now 7.3.0. I also added the new options that allows to reduce the code size a bit. With the empty theme, the build of `tests/pkg_lvgl` is now 91kB (compared to 83kB on master).
By default (with the material theme), the build is 107kB (before it was 111kB).